deliver/lib/deliver/detect_values.rb

```diff
@@ -67,5 +69,17 @@ def find_platform(options)
         options[:platform] = 'osx'
       end
     end
+
+    def verify_languages!(options)
+      languages = options[:languages]
+      return unless languages
+
+      all_languages = Spaceship::Tunes.client.available_languages
+      diff = languages - all_languages
```

Comment:
@taquitos Wait, isn't it good that we aren't doing a case-insensitive compare? We need it to be `en-US`, right? And that is what it does 🤔 

```sh
[!] The following languages are invalid and cannot be activated: en-us

Valid languages are: ["en-CA", "fr-CA", "en-US", "es-MX", "en-GB", "de-DE", "fr-FR", "nl-NL", "el", "tr", "da", "fi", "it", "no", "pt-PT", "ru", "es-ES", "sv", "en-AU", "zh-Hans", "zh-Hant", "id", "ja", "ko", "ms", "th", "vi", "pt-BR"]
```